### PR TITLE
Update OTSessionManager.java - Fix camera lifecycle

### PR DESCRIPTION
**Pull request by cnx-carlosvogt**
### Contributing checklist
- [ x] Code must follow existing styling conventions
- [x ] Added a descriptive commit message
- [ x] Sample apps updated if needed

Bugfix #150 
This PR fixes the problem of the camera freezing in certain situations, such as when another application takes control of the camera and when returning to opentok the camera remains frozen, with this correction this no longer occurs.

### CODE REVIEW

**Comment by cnx-renansoares** [comment]
Hi guys,

We are using the lib for a while and had this problem. We solved this doing what @cnx-carlosvogt mentioned above.

**Review by jvalli — APPROVED** [review]
LGTM

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java [hunk 1]
```diff
@@ -4,14 +4,16 @@
...
  */
 
+import android.os.Build;
 import android.util.Log;
 import android.widget.FrameLayout;
 import android.view.View;
 
 import androidx.annotation.Nullable;
-
+import androidx.annotation.RequiresApi;
```
Comment (jeffswartz) [inline android/src/main/java/com/opentokreactnative/OTSessionManager.java:13]: @cnx-carlosvogt cnx-carlosvogt Looks. I'd like to merge this is. Is RequiresApi used? I don't see it referenced in the code.

Comment (cnx-renansoares) [reply android/src/main/java/com/opentokreactnative/OTSessionManager.java:13]: Hey, @jeffswartz . I work with @cnx-carlosvogt and he is on vacation now. Yes, we can proceed with this deletion.

**Review by marinaserranomontes — APPROVED** [review]
LGTM

**Review by jeffswartz — APPROVED** [review]

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java (hunk 1 above)
Comment (jeffswartz) [reply android/src/main/java/com/opentokreactnative/OTSessionManager.java]: Great. Thanks, @cnx-renansoares. I am merging this in and will create a new version of the library with fix. And thanks again @cnx-carlosvogt for fixing this.

**Pull request merged**